shared/src/utils/math.ts

```diff
@@ -174,3 +175,87 @@ export const amountAfterSlippage = (
     .sub(ethers.utils.parseUnits(slippage.toFixed(3), decimals));
   return num.mul(discountValue).div(BigNumber.from(10).pow(decimals));
 };
+
+export const calculateInitialveRBNAmount = (
```

Comment:
Worth moving this out to utils/veRBNMath.ts because this is concerned with veRBN specific calculations